db/migrate/20230914205521_add_column_and_index_to_listing_images.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnAndIndexToListingImages < ActiveRecord::Migration[6.1]
```

Comment:
@jimlin-sfgov Pinging you on this since you mentioned you have some experience with this. Let me know your thoughts!

Reply:
My only feedback is to make the migration file name more specific by including the name of the added column.